gapis/api/vulkan/api/queue.api

```diff
@@ -164,6 +171,10 @@ cmd VkResult vkQueueSubmit(
   leaveSubcontext()
   execPendingCommands(queue)
   fence
+  if (fence != 0) {
```

Comment:
I am quite surprised this doesn't collide with the fence keyword above.

Reply:
Added some comments for the the next time someone has to read this confusing overloading of 'fence'.